src/init/templates/ossec-hids-suse.init

```diff
@@ -45,22 +45,23 @@ rc_reset
 # considered a success.
 
 # Just to make sure ossec is installed ...
```

Comment:
This comment could be updated, replacing **ossec** for example

Reply:
Fixed